extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -152,10 +152,12 @@ public function stats() {
 	 * @return integer|mixed data of blocks applied or numeric value (0 - block applied, 1 - no block applied)
 	 */
 	private function sendToPhalanxDaemon( $action, $parameters ) {
-		$baseurl = F::app()->wg->PhalanxServiceUrl;
+		global $wgConsulServiceTag, $wgConsulUrl;
+
+		$baseurl = ( new Wikia\Service\Gateway\ConsulUrlProvider( $wgConsulUrl, $wgConsulServiceTag ))->getUrl( 'phalanx' );
```

Comment:
however single request to app is not so long, so maybe it make sense to inject it